src/Plugins/RunawayProcessKiller/RunawayProcessKillerExtension.cs

```diff
@@ -241,10 +241,27 @@ public override void OnWrapperStarted()
                 Logger.Debug("No runaway process with PID=" + pid + ". The process has been already stopped.");
                 return;
             }
+            catch (InvalidOperationException)
```

Comment:
> The process was not started by this object.

The description of this exception is confusing, however, it should not by thrown on local machine.